lab-caleb/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "lab-caleb",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha",
+    "lint": "npm run lint",
```

Comment:
**TODO FOR LOST POINTS**

Good job remembering your lint script, but making it ``npm run lint`` actually causes an endless loop. Reference this lecture code's ``package.json`` to see how it should be formatted.

https://github.com/codefellows/seattle-javascript-401d15/blob/master/class-02-tools-and-context/demos/03-package-json/package.json